index.html

```diff
@@ -5,27 +5,38 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Christian-Hakizimana Portfolio</title>
-    <link rel="stylesheet" type="text/css" href="First_section.css" />
+    <link rel="stylesheet" type="text/css" href="Mobile_version.css" />
     <link rel="stylesheet" media='screen and (min-width: 768px)' href="Desktop.css" />
 
   </head>
 
   <body>
     <section class="head">
-      <header class="toolbar">
-        <a class="nav" href="#">My Logo</a>
-        <nav><img id="menu" src="images/Union.png" alt="navigation" />
+      <header class="toolbar" id="toolbar">
+        <a class="nav nav-logo" href="#">My Logo</a>
+        <nav>
+          <img id="hamburger" class="hamburger" src="images/Union.png" alt="navigation">
           <ul class="menu">
             <li> <a class="me" href="#portfolio">Portfolio</a></li>
             <li> <a  class="me" href="#About">About</a></li>
             <li> <a  class="me" href="#contact">Contact</a></li>
            </ul>
         </nav>
       </header>
+
+      <div class="Popup" id="popup">
+      <img class="cancel" id="cancel" src="images/Icon - Cancel.png" alt="...">
+        <ul class="box">
+          <li> <a class="port" href="#portfolio">Portfolio</a></li>
+          <li> <a  class="Ab" href="#About">About</a></li>
+          <li> <a  class="cont" href="#contact">Contact</a></li>
+         </ul>
```

Comment:
- Great Job implementing the mobile menu. 🚀 

- I'd like to kindly suggest that when the user clicks (or taps) any of the mobile menu options, the correct part of the page is displayed. In your case, the correct option displays but the mobile menu still appears on the screen. Can we fix that?

For example, when I click on the option `Contact` this is what happens 👇🏽 

| Your mobile menu options  |
| :---:   | 
|<img width="415" alt="Screenshot 2023-01-03 at 12 21 29" src="https://user-images.githubusercontent.com/99246359/210348176-dc55ff90-b82c-42c5-bcde-ad4466507d6a.png">  |

⚠️ This should be the case instead `When the user clicks (or taps) any option of the mobile menu, the mobile menu disappears.`

Reply:
thank you for this, i did not notice it. resolved.